src/main/java/ch/hevs/cloudio/endpoint/CloudioEndpoint.java

```diff
@@ -715,13 +741,13 @@ else if("@logsLevel".equals(action)){
                     try{
                         Level log4jLevel = Level.getLevel(logParameter.getLevel());
                         Configurator.setRootLevel(log4jLevel);
-
-                        //put loglevel in mapDB
-                        DB dbPersistenceData = DBMaker.fileDB(PERSISTENCE_FILE).make();
-                        ConcurrentMap map = dbPersistenceData.hashMap(PERSISTENCE_MAP_NAME).createOrOpen();
-                        map.put(PERSISTENCE_LOG_LEVEL, logParameter.getLevel());
-                        dbPersistenceData.close();
-
+                        synchronized (persistenceLock) {
+                            //put loglevel in mapDB
+                            DB dbPersistenceData = DBMaker.fileDB(PERSISTENCE_FILE).make();
```

Comment:
This is probably not the last time a value has to be persisted, so it would make sense to create the methods `void setPersistentProperty(String key, Object value)` and `Object getPersistentProperty(String key, Object defaultValue)` for that functionality.